modules/configuring-maximum-revision-timeout.adoc

```diff
@@ -1,26 +1,21 @@
 // Module included in the following assemblies:
 //
 // * knative-serving/config-applications/configuring-revision-timeouts.adoc
-:_content-type: PROCEDURE
+:_mod-docs-content-type: PROCEDURE
 [id="configuring-maximum-revision-timeout_{context}"]
 = Configuring maximum revision timeout
 
-By seting the maximum revision timeout, you can ensure that no revision can exceed a specific limit.
+By seting the maximum revision timeout, you can ensure that no revision can exceed a specific limit. The value of your maximum revision timeout must not exceed the `terminationGracePeriodSeconds` value of the activator to prevent in-flight requests being disrupted.
 
 .Prerequisites
 
 * You have installed the {ServerlessOperatorName} and Knative Serving.
-* You have cluster administrator permissions on {ocp-product-title}, or cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+* You have cluster administrator permissions on {ocp-product-title}. Alternatively, cluster administrator or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
```

Comment:
With this edit, the second sentence is now a sentence fragment and needs a verb. For example:

"Alternatively, you have cluster administrator or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}."